Library/AzureVirtualMachine.ts

```diff
@@ -0,0 +1,61 @@
+
+import Config = require("./Config");
+import Util = require("./Util");
+import AutoCollectHttpDependencies = require("../AutoCollection/HttpDependencies");
+
+const AIMS_URI = "http://169.254.169.254/metadata/instance/compute";
+const AIMS_API_VERSION = "api-version=2017-12-01";
+const AIMS_FORMAT = "format=json";
+
+class AzureVirtualMachine {
+
+    public isVM: boolean;
```

Comment:
how do you keep this running every 15 mins when it's from VM?

Reply:
I assumed VM info is never updated during runtime, I will update so meta data request is made each time Statsbeat or Heartbeat need the info